src/LogForwarding.js

```diff
@@ -93,6 +93,23 @@ class LogForwarding {
     })
   }
 
+  /**
+   * Set Log Forwarding to New Relic
+   *
+   * @deprecated use `setDestination('new_relic', {...})`
```

Comment:
We shouldn't need this method. Old ones exist just to make sure we don't break backwards compatibility. `setDestination()` should be able to handle everything (used here https://github.com/adobe/aio-cli-plugin-runtime/pull/311/files#diff-243890719f28cb0b8642575ca636169f057c8f1d9773a9bc037d62f6b1ff8f93R21). Is this method used anywhere?

Reply:
I think the method is used in these files:

- https://github.com/adobe/aio-cli-plugin-app/pull/689/files#diff-467e10a49ff9df960441ed6bf8f4910450aca0b8a28c29f9824cf64311d76584

- https://github.com/adobe/aio-cli-plugin-runtime/pull/311/files#diff-467e10a49ff9df960441ed6bf8f4910450aca0b8a28c29f9824cf64311d76584

Should I remove the `setNewRelic()` method then?

Reply:
I don't think it is necessary there. Those are just mocks for tests. Yes. Please, remove this method.